Pipfile

```diff
@@ -34,6 +34,7 @@ gitpython = "*"
 pynacl = "*"
 python-dateutil = "*"
 click = "*"
+discord = "*"
```

Comment:
Is this necessary?

Reply:
Nope, can't recall why it's there. Maybe when I tried to install a package.
Removing it.